services/web/server/src/simcore_service_webserver/clusters/models.py

```diff
@@ -22,6 +22,32 @@ class Config:
 CLUSTER_NO_RIGHTS = ClusterAccessRights(read=False, write=False, delete=False)
 
 
+class Authentication(BaseModel):
+    type: str
+
+    class Config:
+        extra = Extra.forbid
+
+
+class SimpleAuthentication(Authentication):
+    type: Literal["simple"] = "simple"
+    username: str
+    password: str
```

Comment:
Use ``SecretStr`` https://pydantic-docs.helpmanual.io/usage/types/#secret-types

Reply:
no this creates too much hassle everywhere.